packages/manager/modules/cloud-connect/src/cloud-connect.service.js

```diff
@@ -507,4 +507,22 @@ export default class CloudConnectService {
       CloudConnectService.clearCache(cacheName);
     });
   }
+
+  loadStatistics(cloudConnectId, interfaceId, type, period) {
+    const options = {
+      period,
+      type,
+    };
+
+    return this.$http
+      .get(
+        `/ovhCloudConnect/${cloudConnectId}/interface/${interfaceId}/statistics`,
+        { params: options },
+      )
+      .then((statistics) => {
+        const stats = statistics.data || [];
+        return stats.map((data) => [data.timestamp * 1000, data.value]);
```

Comment:
suggestion
```suggestion
      .then((statistics) => {
        const stats = statistics.data || [];
        return stats.map(({timestamp, value}) => [timestamp * 1000, value]);
```
Also wouldn't it be easier to manipulate as an object instead of an array ?